src/pyhf/infer/calculators.py

```diff
@@ -314,6 +368,12 @@ def teststatistic(self, poi_test):
             >>> asymptotic_calculator.teststatistic(mu_test)
             array(0.14043184)
 
+            Access the best-fit parameters afterwards:
+            (here: :math:`\hat{\mu}` and :math:`\hat{\theta}` fitted to the asimov dataset):
+
+            >>> asymptotic_calculator.fitted_pars.free_fit_to_asimov
```

Comment:
add in `>>> asymptotic_calculator.fitted_pars` so people can be assured it's a `HypoTestFitResult` object.

Reply:
Done